include/addressatcommand.h

```diff
@@ -66,6 +66,28 @@ class AddressAtCommand : public NonKeyedCommand
         *            type.
         */
         virtual Command* clone();
+
+        /*!
+        *   \brief Returns host of database node
+        *   \param address The address of the database node
+        *   \returns The host of the database node
+        */
+        inline std::string parse_host(std::string address)
+        {
+            return address.substr(0, address.find(":"));
+        }
+
+        /*!
+        *   \brief Returns port of database node
+        *   \param address The address of the database node
+        *   \returns The port of the database node
+        */
+        inline uint64_t parse_port(std::string address)
+        {
+            return std::stoul(address.substr(address.find(":") + 1),
```

Comment:
Similar comment as ``parse_host`` except the additional check suggestion would be to make sure ``:`` exists and isn't the last position so we don't return empty string.